server.js

```diff
@@ -91,4 +91,12 @@ app.post('/api/retweet/:id', (req, res) => {
     .catch(error => res.send(error));
 });
 
+app.post('/api/senddirectmessage', (req, res) => {
+  const data = {"event": {"type": "message_create", "message_create": {"target": {"recipient_id": "1121202912946819072"}, "message_data": {"text": req.body.message}}}};
```

Comment:
can you make recipient_id parameterized ? we can use this when we actually integrate with sendDirectMessages inline UI